src/app/modules/shared/time-range-form/time-range.component.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { ToastrService, IndividualConfig } from 'ngx-toastr';
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import { TimeRangeFormComponent } from './time-range-form.component';
+import { EntryState } from '../../time-clock/store/entry.reducer';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { InputDateComponent } from '../../shared/components/input-date/input-date.component';
+import * as entryActions from '../../time-clock/store/entry.actions';
+import * as moment from 'moment';
+
+describe('Reports Page', () => {
+  describe('TimeRangeFormComponent', () => {
+    let component: TimeRangeFormComponent;
+    let fixture: ComponentFixture<TimeRangeFormComponent>;
+    let store: MockStore<EntryState>;
+    const toastrServiceStub = {
+      error: (message?: string, title?: string, override?: Partial<IndividualConfig>) => { }
+    };
+
+    const timeEntry = {
```

Comment:
why not take advantage of TypeScript and use types here?

Reply:
@rene-enriquez-ct per example 

const timeEntry: any = {